src/trap-caching.ts

```diff
@@ -216,6 +216,7 @@ export async function getTotalCacheSize(
   trapCaches: Partial<Record<Language, string>>,
```

Comment:
```suggestion
  trapCaches?: Partial<Record<Language, string>>,
```

Minor: Not sure if you need `Partial` here. This might be fine, too:

```suggestion
  trapCaches?: Record<Language, string>,
```

Reply:
As above, I think the type should record that it is not okay for this to be `undefined`, so that normal production code that doesn't try and trick the compiler can't set this as `undefined`.